### PR TITLE
Begin every comment with a space

### PR DESCRIPTION
**Pull request by lupoDharkael**
This makes the comments more visually pleasing.
Maybe we could define a base style for comments and add it to the documentation?

### CODE REVIEW

**Comment by lupoDharkael** [comment]
The amount of lines touched by this PR is considerable. This may break some PRs and I undertand if it's seen as a superfluous change.

**Comment by sampengilly** [comment]
Is there a rule for this in Clang? It would probably make more sense to apply a new rule and have the changes get picked up over time as files are worked on.

**Comment by pycbouh** [comment]
What sampengilly said. Also, comments should start with a capital letter and end with a full stop.

**Comment by Calinou** [comment]
See also #35998 (which needs a rebase).

**Comment by ThakeeNathees** [comment]
it should be like this IMO
```c+
// A comment.
//commented_out_code();
```

**Comment by lupoDharkael** [comment]
@ThakeeNathees  I guess that could be done matching commented lines ending with `;` but that might not cover all the cases. I don't know how much can the change be automated and the amount of comments is quite big.

**Comment by lupoDharkael** [comment]
@Calinou  Yeah  #35998 would be good to be done before this as it would simplify the work required for this PR. I guess I'll leave this in standby as there is no priority for a change like this and  #35998 would be a nice PR to have merged before working on comment correctness.

File: core/input/input.cpp [hunk 1]
```diff
@@ -821,7 +821,7 @@ void Input::joy_button(int p_device, int p_button, bool p_pressed) {
 
 	_THREAD_SAFE_METHOD_;
 	Joypad &joy = joy_names[p_device];
-	//printf("got button %i, mapping is %i\n", p_button, joy.mapping);
+	// printf("got button %i, mapping is %i\n", p_button, joy.mapping);
```
Comment (aaronfranke) [inline core/input/input.cpp:824]: This is wrong. Commented out code should not start with a space.

While we don't have a complete C++ style guide yet, most of what is in the [GDScript style guide](https://docs.godotengine.org/en/latest/getting_started/scripting/gdscript/gdscript_styleguide.html) that can apply to C++ does apply to the C++ code.

**Review by aaronfranke — CHANGES_REQUESTED** [review]
Might be worth reformatting all of the comments at once such that they start with a capital letter and end with a period.

This needs to be done by hand, it can't be automated easily.

**Comment by aaronfranke** [comment]
Also, it might make more sense to do this incrementally, rather than throughout the entire codebase at once. For example, one user could make a PR for all of the files in `core/`, and another user could make a PR for `editor/`, etc.

File: core/input/input.cpp (hunk 1 above)
Comment (lupoDharkael) [reply core/input/input.cpp:824]: I used a simple regex to replace the code, I'll change that after #35998 is reworked.

**Comment by lupoDharkael** [comment]
@aaronfranke that might be the best approach to reduce the impact of PR conflict.

**Comment by IAmActuallyCthulhu** [comment]
Relates to #31261, at this time idk if I will finish it ¯\_(ツ)_/¯.

**Comment by YuriSizov** [comment]
This PR has been inactive for quite some time, and it has a massive impact. I believe we want this, but ideally we want to make sure our comment formatting rules are preserved and new PRs don't break them. Not sure if clang-format allows this or not.

Either way, this PR as it stands needs a massive rebase, and it needs to comply with the rest of the comment formatting rules, such as starting with a capital letter and ending with a period. And if it's a code line that is commented, it should not be affected at all. If we are doing a pass like that then we might as well do it right.

So I suggest remaking this anew, and use the approach @aaronfranke suggested, doing it in batches.

**Pull request closed without merging**